server/webhook_parser.go

```diff
@@ -108,33 +111,45 @@ func parseWebhookChangeLog(jwh *JiraWebhook) Webhook {
 		case field == "status":
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedStatus, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedStatus, field, fieldId, fromWithDefault, toWithDefault)
 		case field == "priority":
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedPriority, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedPriority, field, fieldId, fromWithDefault, toWithDefault)
 		case field == "summary":
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedSummary, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedSummary, field, fieldId, fromWithDefault, toWithDefault)
 		case field == "description":
-			events = append(events, parseWebhookUpdatedDescription(jwh, from, to))
+			event = parseWebhookUpdatedDescription(jwh, from, to)
 		case field == "Sprint" && len(to) > 0:
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedSprint, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedSprint, field, fieldId, fromWithDefault, toWithDefault)
 		case field == "Rank" && len(to) > 0:
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedRank, field, strings.ToLower(fromWithDefault), strings.ToLower(toWithDefault)))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedRank, field, fieldId, strings.ToLower(fromWithDefault), strings.ToLower(toWithDefault))
 		case field == "Attachment":
-			events = append(events, parseWebhookUpdatedAttachments(jwh, from, to))
+			event = parseWebhookUpdatedAttachments(jwh, from, to)
 		case field == "labels":
-			events = append(events, parseWebhookUpdatedLabels(jwh, from, to, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedLabels(jwh, from, to, fromWithDefault, toWithDefault)
 		case field == "assignee":
-			events = append(events, parseWebhookAssigned(jwh, from, to))
+			event = parseWebhookAssigned(jwh, from, to)
 		case field == "issuetype":
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedIssuetype, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedIssuetype, field, fieldId, fromWithDefault, toWithDefault)
+		case field == "Fix Version":
+			event = parseWebhookUpdatedField(jwh, eventUpdatedFixVersion, field, fieldId, from, to)
 		case field == "reporter":
-			events = append(events, parseWebhookUpdatedField(jwh, eventUpdatedReporter, field, fromWithDefault, toWithDefault))
+			event = parseWebhookUpdatedField(jwh, eventUpdatedReporter, field, fieldId, fromWithDefault, toWithDefault)
+		case strings.HasPrefix(fieldId, "customfield_"):
+			eventType := fmt.Sprintf("event_updated_%s", fieldId)
+			event = parseWebhookUpdatedField(jwh, eventType, field, fieldId, from, to)
+		}
+
+		if event != nil {
+			event.eventTypes.Add(eventUpdatedAll)
```

Comment:
It's not really "All" it's more like "any". It also seems strange to have this separate when we have that intersection function.

Reply:
I agree, "any" makes much more sense. Also, it does make more sense to derive that the "any" user-choice matches up explicitly, rather than doing it the way shown here. The solution here will lead to bugs.